server/src/main/java/org/spine3/server/entity/RepositoryView.java

```diff
@@ -27,6 +27,13 @@
 /**
  * A view on a repository.
  *
+ * <p>A {@link Repository} may have a view that provides a different set of entities.
+ * For example, there can be views that return archived or deleted entities (that are
+ * not visible by default).
+ *
+ * <p>The default behaviour of loading entities in a {@code Repository} is also a view,
```

Comment:
`The behaviour ... is a view` seems to complex to read. Something is missing in the sentence.